lib/router.js

```diff
@@ -74,6 +75,35 @@ Router.prototype._buildPath = function route(node, path, value) {
     return node;
 };
 
+/**
+ * Tries to require the module supplied in the configuration
+ */
+Router.prototype._requireModule = function(modName, opts) {
+    var self = this;
+    opts = opts || { mod: modName, baseTried: false, modsTried: false };
+    try {
+        return require(modName);
+    } catch (e) {
+        if (/^\//.test(opts.mod) || (opts.baseTried && opts.modsTried)) {
+            // we have a full path here which can't be required, or we tried
+            // all of the possible combinations, so bail out
+            e.moduleName = opts.mod;
+            throw e;
+        } else {
+            // This might be a relative path, convert it to absolute and try again
+            if (!opts.baseTried) {
+                // first, try to load it from the app's base path
+                opts.baseTried = true;
+                modName = path.resolve(self._options.appBasePath + '/' + opts.mod);
```

Comment:
Nit: We could use `path.join` here to avoid manually adding the `/`, will be a little nicer.